front/src/components/member/MemberUpdateModal.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="d-flex flex-column modal-mask" @click.self="closeModal">
+    <VSpacer />
+    <transition name="bounce">
+      <div v-if="rendered" class="pa-6 pb-1 modal-container">
+        <VTextField
+          label="새 닉네임을 입력하세요."
+          color="grey darken-2"
+          v-model="newNickname"
+          @keypress.enter="updateMember"
+        />
+        <div class="button-box">
+          <v-btn
+            text
+            class="mb-2 text-subtitle-1 font-weight-bold grey--text text--darken-1"
+            @click.prevent="closeModal"
+          >
+            취소
+          </v-btn>
```

Comment:
```suggestion
    <transition name="bounce" @after-leave="close">
      <div v-if="rendered" class="pa-6 pb-1 modal-container">
        <VTextField
          label="새 닉네임을 입력하세요."
          color="grey darken-2"
          v-model="newNickname"
          @keypress.enter="updateMember"
        />
        <div class="button-box">
          <v-btn
            text
            class="mb-2 text-subtitle-1 font-weight-bold grey--text text--darken-1"
            @click="bounceOut"
          >
            취소
          </v-btn>
```

```javascript
    bounceOut() {
      this.rendered = false;
    },
```

이렇게 주면 모달창을 종료할 때도 트랜지션이 가능해요!
그리고 submit 상황이 아니라면 prevent는 필요하지 않아요.